pages/library/_viewId.vue

```diff
@@ -132,7 +132,7 @@ export default Vue.extend({
 
       if (
         this.collectionInfo &&
-        ['CollectionFolder', 'Folder', 'UserView'].includes(
+        ['CollectionFolder', 'Folder', 'UserView', 'playlists'].includes(
```

Comment:
AFAIK, a collection can't have type playlists?

Or at least my playlists collection doesnt have type 'playlists'

Reply:
@Camc314 I'm not sure to be honest, playlists doesn't load for me in master and with this line here pointed out by @MrTimscampi it works